pkg/controller/change/device/computedelta_test.go

```diff
@@ -16,10 +16,10 @@ package device
 
 import (
 	"fmt"
+	"github.com/onosproject/onos-config/api/types"
+	"github.com/onosproject/onos-config/api/types/change"
+	devicechangetypes "github.com/onosproject/onos-config/api/types/change/device"
```

Comment:
This is why `github.com/onosproject/onos-config/api/types/change/device` was given the alias `devicechangetypes` consistently through out the code. `devicechanges` had been used for the package `github.com/onosproject/onos-config/pkg/types/change/device`

I don't care what alias is used for either as long as the usage is consistent throughout the project. It includes code readability and consistency